React/training/src/app/screens/Game/index.js

```diff
@@ -1,46 +1,33 @@
 import React, { Component } from 'react';
 import Board from './components/Board';
 import styles from './styles.module.scss';
+import { connect } from 'react-redux';
+import gameActions from '../../../redux/Game/actions';
 
 class Game extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      history: [{
-        squares: Array(9).fill(null),
-      }],
-      stepNumber: 0,
-      xIsNext: true,
-    };
-  }
-
   handleClick(i) {
```

Comment:
Handle click should be an `action`. So it should be inside `src/redux/game/actions.js`